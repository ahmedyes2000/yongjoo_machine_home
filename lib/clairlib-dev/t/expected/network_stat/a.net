*network t/docs/gg2/produced/a
*vertices 21
1 "6"
2 "12"
3 "9"
4 "7"
5 "22"
6 "19"
7 "11"
8 "24"
9 "13"
10 "25"
11 "18"
12 "4"
13 "1"
14 "31"
15 "29"
16 "20"
17 "15"
18 "23"
19 "14"
20 "17"
21 "27"
*arcs
1 16 1
1 20 1
1 13 1
1 8 1
1 5 1
1 18 1
1 9 1
2 11 1
2 9 1
2 12 1
2 21 1
2 17 1
2 18 1
2 4 1
2 10 1
3 6 1
4 12 1
4 18 1
4 11 1
4 17 1
4 21 1
4 10 1
4 2 1
5 1 1
5 7 1
6 3 1
7 5 1
8 1 1
8 16 1
9 1 1
9 15 1
9 2 1
10 18 1
10 2 1
10 12 1
10 21 1
10 4 1
10 17 1
10 11 1
11 17 1
11 10 1
11 21 1
11 12 1
11 2 1
11 18 1
11 4 1
12 4 1
12 11 1
12 2 1
12 10 1
12 13 1
12 21 1
12 18 1
12 17 1
13 1 1
13 12 1
14 19 1
15 9 1
16 1 1
16 8 1
17 12 1
17 4 1
17 11 1
17 18 1
17 2 1
17 21 1
17 10 1
18 10 1
18 1 1
18 17 1
18 12 1
18 21 1
18 2 1
18 11 1
18 4 1
19 14 1
20 1 1
21 11 1
21 4 1
21 10 1
21 2 1
21 12 1
21 18 1
21 17 1
