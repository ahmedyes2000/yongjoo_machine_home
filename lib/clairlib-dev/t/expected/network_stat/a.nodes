16
12
22
4
27
21
14
11
28
32
31
20
6
23
19
24
8
2
10
7
18
13
9
29
25
1
26
15
3
30
5
17
