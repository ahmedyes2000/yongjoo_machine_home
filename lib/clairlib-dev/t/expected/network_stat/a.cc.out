reading the input...
done!
clustering coefficient for 11 is 0
clustering coefficient for 7 is 1
clustering coefficient for 17 is 0
clustering coefficient for 22 is 0
clustering coefficient for 1 is 0
clustering coefficient for 18 is 1
clustering coefficient for 23 is 0.75
clustering coefficient for 13 is 0
clustering coefficient for 29 is 0
clustering coefficient for 27 is 1
clustering coefficient for 6 is 0.0476190476190476
clustering coefficient for 25 is 1
clustering coefficient for 9 is 0
clustering coefficient for 12 is 0.75
clustering coefficient for 14 is 0
clustering coefficient for 20 is 1
clustering coefficient for 15 is 1
clustering coefficient for 4 is 0.75
clustering coefficient for 24 is 1
clustering coefficient for 19 is 0
clustering coefficient for 31 is 0

number of nodes skipped (with degree > 5000): 0
